src/sql/workbench/browser/modal/modal.ts

```diff
@@ -234,7 +235,7 @@ export abstract class Modal extends Disposable implements IThemable {
 		const modalBodyClass = (this._modalOptions.isAngular === false ? 'modal-body' : 'modal-body-and-footer');
 
 		this._modalBodySection = DOM.append(modalContent, DOM.$(`.${modalBodyClass}`));
-		this.renderBody(this._modalBodySection);
+		this._dialogPaneBody = <HTMLElement>this.renderBody(this._modalBodySection);
```

Comment:
why don't we just use `this._modalBodySection` anywhere we are want to search the body?

Reply:
yeah I tried that first - but the element return its not dialog pane